src/site/content/en/blog/aem-with-workbox/index.md

```diff
@@ -0,0 +1,138 @@
+
+{% Img src="image/sQ51XsLqKMgSQMCZjIN0B7hlBO02/j97xB2WYaExyh8SbI3pw.png", alt="What is a service worker?", width="800", height="369" %}
+
+{% Img src="image/sQ51XsLqKMgSQMCZjIN0B7hlBO02/4izQ6TMiE9PHdRTor7b2.png", alt="Workbox Logo", width="200", height="47" %}
+
+Service workers can be hard to write from scratch. Workbox was created to make it easier. Workbox is a set of libraries to help you write and manage [service workers](https://developers.google.com/web/fundamentals/primers/service-workers) and caching with the [Cache Storage API](/cache-api-quick-guide/). Service workers and the Cache Storage API, when used together, control how assets (HTML, CSS, JS, images, etc.) are requested from the network or cache, even allowing you to return cached content when offline. With Workbox, you can quickly set up and manage both, and more, with production-ready code.
+
+## Upgrading an AEM site to a PWA
+
+[Adobe Experience Manager](https://business.adobe.com/products/experience-manager/sites/aem-sites ) (AEM) is a comprehensive content management solution for building websites, mobile apps, forms, and digital signage. It makes it easy to manage your marketing content and assets.
+
+While AEM provides a rich library for building web applications, until now it's been difficult to build a PWA by adding a service worker and a manifest.
+
+Adobe Experience Manager Sites is the UI building tool that is part of Adobe Experience Manager. When used with Adobe Experience Manager as a cloud service, AEM Sites makes it easy to convert any existing AEM website or single page application into an installable offline-enabled Progressive Web App with just configuration and no coding. It uses Workbox to deliver the best practices for Progressive Web Apps and abstracts the complexities of writing boilerplate manifests and service workers.
+
+AEM supports localization of content which means you can have different branding and even different offline content for different locales. To do this build different PWA configurations for each language master.
+
+### Getting started with PWA configuration on AEM
+
+Log into Adobe Experience Manager as a Cloud Service and select any Adobe Experience Manager Sites page or language master and click properties. You should see a tab called Progressive Web App. (Note: if you do not see this tab, please contact Adobe to enable this feature.) You can configure the installation and the look and feel of your Progressive Web Apps with just a few clicks.
```

Comment:
```suggestion
Log into Adobe Experience Manager as a Cloud Service and select any Adobe Experience Manager Sites page or language master and click properties. You should see a tab called Progressive Web App. You can configure the installation and the look and feel of your Progressive Web Apps with just a few clicks.

{% Aside %}
If you don't see the Progressive Web App tab in Adobe Experience Manager, please contact Adobe to enable this feature.
{% endAside %}
```

Reply:
This feels like a good thing to break out into an aside, since it's, well, an aside within the paragraph.